runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -10176,10 +10170,19 @@ roundArrayLengthToObjectAlignment(TR::CodeGenerator* cg, TR::Node* node, TR::Ins
       }
    }
 
+static void
+genMemoryZeroingLoop(TR::CodeGenerator* cg, TR::Node* node, TR::Instruction*& iCursor, TR::Register * addressReg, TR::Register * loopCountRegister)
+   {
+   TR::LabelSymbol * loopTopLabel = generateLabelSymbol(cg);
```

Comment:
Instead of `loopTopLabel`, I thin, startLabel/loopStartLabel would be more meaningful here.